brave/browser-laptop
# 0.13.1: Payment history footer message needs correct past/future context

Fixes #6059

Auditor: @bsclifton 

Test Plan:

1. ensure that `reconcileStamp` in `ledger-state.json` is in the future - the message in about:preferences#payments should show a future contribution.

2. set `reconcileStamp` to less than a day in the past - the message should indicate today.

3. set `reconcileStamp` to more than a day in the past - the message should indicate overdue.
